content/guides/http/go/net_http/server.md

```diff
@@ -0,0 +1,226 @@
+---
+title: "Server"
+date: 2018-11-27T01:12:31-07:00
+draft: false
+aliases: [/integrations/http/go/net_http/server]
+weight: 2
+logo: /images/gopher.png
+---
+
+- [Introduction](#introduction)
+- [Traces](#traces)
+- [Metrics](#metrics)
+- [End-to-end example](#end-to-end-example)
+- [Viewing traces](#view-traces)
+- [Viewing metrics](#view-metrics)
+- [References](#references)
+
+### Introduction
+
+OpenCensus provides a package [go.opencensus.io/plugin/ochttp](https://godoc.org/go.opencensus.io/plugin/ochttp) which has a custom HTTP handler [ochttp.Handler](https://godoc.org/go.opencensus.io/plugin/ochttp#Handler) that can wrap your [HTTP handlers](https://golang.org/pkg/net/http/#Handler) while running your [HTTP servers](https://golang.org/pkg/net/http/#Server). For example
+
+```go
+package main
+
+import (
+	"log"
+	"net/http"
+
+	"go.opencensus.io/plugin/ochttp"
+)
+
+func main() {
+	originalHandler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("Hello, World!"))
+	})
+	och := &ochttp.Handler{
+		Handler: originalHandler, // The handler you'd have used originally
+	}
+
+	// Now use the instrumnted handler
+	if err := http.ListenAndServe(":9999", och); err != nil {
+		log.Fatalf("Failed to run the server: %v", err)
+	}
+}
+```
+
+### Traces
+The instrumented handler creates a trace automatically for each request that is received. We shall need a [Trace exporter](/exporters/supported-exporters/go) to examine the traces.
```

Comment:
Just like with the client example I have a minor issue with trace exporter being needed to examine traces (omitting the step that the exporter sends it to a tracing ecosystem for inspection)

How about:
We shall need a trace exporter to send the data to a backend, allowing us to examine the traces?